pkg/validate/container.go

```diff
@@ -173,9 +224,11 @@ func getContainerStatus(c internalapi.RuntimeService, containerID string) *runti
 }
 
 // verifyContainerStatus verifies whether container status for given containerID matches.
-func verifyContainerStatus(c internalapi.RuntimeService, containerID string, expectedStatus runtimeapi.ContainerState, stateName string) {
+func verifyContainerStatus(c internalapi.RuntimeService, containerID string, expectedStatus runtimeapi.ContainerState, stateName string) bool {
 	status := getContainerStatus(c, containerID)
 	Expect(status.State).To(Equal(expectedStatus), "Container state should be %s", stateName)
```

Comment:
so, what should use here? Just compare the container state with expectedStatus directly?